scripts/protocgen.sh

```diff
@@ -1,39 +1,46 @@
 #!/usr/bin/env bash
+set -euo pipefail
 
-set -eo pipefail
+# By default, this script runs against the latest commit to the master branch
+# in the Tendermint spec repository. To use this script with a different version
+# of the spec repository, run it with the $VERS environment variable set to the
+# desired branch name or commit hash from the spec repo.
 
 : ${VERS:=master}
-URL_PATH=archive/
-if [[ VERS -ne master ]]; then
-    URL_PATH=archive/refs/tags/v
-fi
+URL_PATH=tarball/
```

Comment:
Can we inline this, since it doesn't change?

Reply:
done!